LICENSE

```diff
@@ -1,21 +1,27 @@
-MIT License
+Copyright (c) 2022 to 2024 inclusive, QuTiP developers and contributors.
```

Comment:
I would suggest to add the following text as first line, like the orginsl github template for this license

```
BSD 3-Clause License
```

Reply:
Okay~ Done